app/scripts/directives/createSecret.js

```diff
@@ -117,38 +117,38 @@ angular.module("openshiftConsole")
             data: {}
           };
 
+          var secretData = {};
           switch (authType) {
             case "kubernetes.io/basic-auth":
               // If the password/token is not entered either .gitconfig or ca.crt has to be provided
               if (data.passwordToken) {
-                secret.data = {password: window.btoa(data.passwordToken)};
+                secretData['password'] = data.passwordToken;
```

Comment:
We typically don't use the bracket `[]` notation unless the key has special characters.

```js
secretData.password = data.passwordToken;
```